packages/web-app-external/src/App.vue

```diff
@@ -132,12 +149,49 @@ export default defineComponent({
       }
     }).restartable()
 
+    // switch to write mode when edit is clicked
+    const catchClickMicrosoftEdit = (event) => {
+      try {
+        if (JSON.parse(event.data)?.MessageId === 'UI_Edit') {
+          loadAppUrl.perform('write')
+        }
+      } catch (e) {}
+    }
+    watch(
+      applicationName,
+      (newAppName, oldAppName) => {
+        console.log('appNames', newAppName, oldAppName)
+        if (newAppName === 'Office365' && newAppName !== oldAppName) {
```

Comment:
can this be configured or is this hardcoded in the app provider? I really don't know ... 

@micbar @kulmann ?

Reply:
For the record (you already know long time, but I'd like to write it down): it's configurable in the app provider.